plugins/initialization-control/src/main/java/org/qbicc/plugin/initializationcontrol/QbiccFeatureTypeBuilder.java

```diff
@@ -129,6 +133,15 @@ public FieldElement resolveField(int index, DefinedTypeDefinition enclosing, Fie
         }
     }
 
+    @Override
+    public DefinedTypeDefinition build() {
+        DefinedTypeDefinition result = delegate.build();
+        if (reflectiveClass) {
+            ReachabilityRoots.get(classContext.getCompilationContext()).registerReflectiveClass(result.load());
```

Comment:
forcing loading here is a bit suspect, but a class is only going to be reflective if it was tagged in a qbicc-feature, so it seems to work out ok.